run_ct_local.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+ct install --config ct.yaml \
```

Comment:
is this different from how lint-test workflow calls this? If not, I would prefer to remove this shell script, add a Makefile with a target `test` containing this exact command, and change the workflow to call `make test-es`

Reply:
https://github.com/jaegertracing/helm-charts/pull/698